cmd/svcat/binding/unbind_cmd.go

```diff
@@ -83,6 +105,43 @@ func (c *unbindCmd) Run() error {
 	var bindings []types.NamespacedName
 	var err error
 
+	if c.abandon {
+		fmt.Fprintln(c.Output, "This action is not reversible and may cause you to be charged for the broker resources that are abandoned.")
+		if !c.skipPrompt {
+			fmt.Fprintln(c.Output, "Are you sure? [y|n]: ")
+			s := bufio.NewScanner(os.Stdin)
+			s.Scan()
+
+			err = s.Err()
+			fmt.Fprintln(c.Output, err)
+
+			if strings.ToLower(s.Text()) != "y" {
+				err = fmt.Errorf("aborted abandon operation")
+				return err
+			}
+		}
+
+		var removedBindings []types.NamespacedName
+		if c.instanceName != "" {
+			si := &v1beta1.ServiceInstance{ObjectMeta: metav1.ObjectMeta{Name: c.instanceName, Namespace: c.Namespace}}
+			removedBindings, err = c.App.RemoveFinalizersByInstance(si)
+		} else {
+			retrievedBindings := c.getBindingsToDelete()
+			removedBindings, err = c.App.RemoveFinalizersForBindings(retrievedBindings)
+		}
+
+		if err != nil {
+			return err
+		}
+		for _, binding := range removedBindings {
+			fmt.Fprintln(c.Output, fmt.Sprintf("deleted %s", binding.Name))
+		}
+		if c.instanceName != "" {
+			fmt.Fprintln(c.Output, fmt.Sprintf("deleted %s", c.instanceName))
+		}
+		return nil
```

Comment:
this return statement needs to be removed - otherwise we just remove the finalizers and don't actually delete the binding